omegatrade-app/backend/app/config/database.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+const {Spanner} = require('@google-cloud/spanner');
+// Creates Spanner client
+const spanner = new Spanner({projectId: process.env.PROJECTID,});
+// Initialize Spanner instance
+const instance = spanner.instance(process.env.INSTANCE);  
+// Environment variable assigned here
+const databaseId = process.env.DATABASE;
+// Initialize database
+const database = instance.database(databaseId,{
+    idlesAfter: 10,
+    keepAlive: 30,
```

Comment:
What is the reason that you are not using the default session pool configuration? I would recommend adding comments for any deviations from the defaults that explains __why__ the default value is not the best option for the application. Also, I would leave out the configuration options that are equal to the default (e.g. `label: {}`)

Reply:
yes, correct there is no reason for changing the default session pool configuration.

Reply:
changed to default session pool configuration.